### PR TITLE
Merge main into dev staging

### PR DESCRIPTION
**Pull request by biosafetylvl5**
No major changes to code, just merging existing work in `dev-staging` and `main`. 

Minor changes:
 - Update to dockerfile for CI (path of `$GEOIPS_PACKAGES_DIR`)
 - Moved `releases/upcoming/*` to `releases/1.14.4/*` because that should have happened during the release

### CODE REVIEW

**Comment by biosafetylvl5** [comment]
This will fail the note edits check because the release failed that check

**Comment by biosafetylvl5** [comment]
Docs build fine; issue is with CI not updated for bug fix in build docs - see https://github.com/NRLMMD-GEOIPS/geoips_ci/pull/8 for fix

**Comment by biosafetylvl5** [comment]
This is passing full test, except for the geotiff test which is also failing on main and dev-staging

File: docs/source/releases/1.14.3/update-geoips-clavrx-test-data.yaml
Comment (mindyls) [inline docs/source/releases/1.14.3/update-geoips-clavrx-test-data.yaml:1]: I think all the latest release notes should stay in latest ? We're these already included in a merge, or is this the new functionality on dev stagimg?

**Review by mindyls — COMMENTED** [review]
Note I moved the 1.14.4-internal-release.yaml to 1.14.4 (was originally in upcoming, then latest, now 1.14.4. I think it should be in 1.14.4)

**Review by mindyls — APPROVED** [review]

File: docs/source/releases/1.14.3/update-geoips-clavrx-test-data.yaml
Comment (jsolbrig) [reply docs/source/releases/1.14.3/update-geoips-clavrx-test-data.yaml:1]: Yes, this is just a gigantic review of things that we've already reviewed, approved, and merged.

Comment (biosafetylvl5) [reply docs/source/releases/1.14.3/update-geoips-clavrx-test-data.yaml:1]: Yes, this is just fixing all the merge conflicts

**Review by jsolbrig — DISMISSED** [review]

**Review by jsolbrig — APPROVED** [review]

**Pull request merged**